docs/ABI.md

```diff
@@ -0,0 +1,1176 @@
+orphan
+
```

Comment:
@paulyoung I think the `orphan` directive can just be removed.

When Sphinx builds the RST files into HTML, it emits a warning if it finds any files that don’t appear in the table of contents. The `:orphan:` directive suppresses this warning. See [Miscellaneous markup](http://sphinx-doc.org/markup/misc.html#metadata).